NOTICE.md

```diff
@@ -1,8 +1,14 @@
 HASH: 152b6b858d2f6b94fa398fa4545d5e130ab24434
 #############################################################
-WARNING: This update bumps to a pre-release rpi-4.19.y linux
-tree and should only be used if there is a specific reason to 
-do so, for example, a request by a Raspberry Pi engineer.
+WARNING: 'rpi-update' updates to pre-releases of the linux 
+kernel tree and Videocore firmware.
+
+'rpi-update' should only be used if there is a specific 
+reason to do so. For example, a request by a Raspberry Pi 
+engineer.
+
+DO NOT use 'rpi-update' as part of a regular update process.
+
 Be aware there could be compatibility issues with some drivers
 Discussion here:
 https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=224931
```

Comment:
Can you remove these 3 lines - again  they are specifically referring to 4.19 issues before 4.19 was the stable kernel.

Reply:
No problem, was not aware that they were specific.